src/rust/engine/options/src/args.rs

```diff
@@ -80,11 +77,7 @@ impl Args {
         Ok(None)
     }
 
-    fn get_list<T>(
-        &self,
-        id: &OptionId,
-        parse_list: fn(&str) -> Result<Vec<ListEdit<T>>, ParseError>,
-    ) -> Result<Option<Vec<ListEdit<T>>>, String> {
+    fn get_list<T: ListMember>(&self, id: &OptionId) -> Result<Option<Vec<ListEdit<T>>>, String> {
```

Comment:
No more need for the `parse_list` function pointer - `ListMember` has the functionality we need (via its `Parseable` supertrait).